roles/x509-identity-mgmt/templates/x509_identity_mgmt_cluster.yaml.j2

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+
+metadata:
+  name: x509-identity-mgmt
+  namespace: {{ dojot_namespace }}
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: NotIn
+            values:
+            - verne-kafka
+  selector:
+    matchLabels:
+      app: x509-identity-mgmt
+  replicas: 1
+  template: #templates for the pods
+    metadata:
+      labels:
+        app: x509-identity-mgmt
```

Comment:
In the Ansible I'm running for the 100K epic, I changed the tags to something like this:
```yaml
app: dojot
name: x509-identity-mgmt
```

This way, we can filter all logs for Dojot, having a similar effect as docker-compose logs. This behaviour is useful for some tests, as @betefaber said to me.
Don't forget to change the selector in the service file.